### PR TITLE
third attempt at unblacklisting prosic2

### PR DESCRIPTION
**Pull request by dlaehnemann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/prosic/meta.yaml [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+  url: https://github.com/varlociraptor/prosic2/archive/v{{ version }}.tar.gz
+  sha256: c5490684d04bb4a9bb47de39c9d171f775f6d3ffa1c9337df7af9d8774d33974
 
 requirements:
   build:
     - {{ compiler('c') }}
     - rust >=1.30
     - pkg-config #[osx]
+    # needed, otherwise build errors with `./htslib/htslib/vcf.h:38:10: fatal error: 'assert.h' file not found`
+    - clangdev
```
[the remark this answers is not in the document]
Comment (dlaehnemann) [reply recipes/prosic/meta.yaml]: Nope, the build error was raised on the linux-test. Should I add that to the comment?

[the remark this answers is not in the document]
Comment (dlaehnemann) [reply recipes/prosic/meta.yaml]: And OSX was not run, while linux-test failed -- so no clue if this is maybe linux only.

[the remark this answers is not in the document]
Comment (dpryan79) [reply recipes/prosic/meta.yaml]: Weird, `assert.h` exists by default on linux and clang isn't used at all. Apparently this is invoking some byproduct that's solving the actual problem, which is fine at the end of the day.

[the remark this answers is not in the document]
Comment (dlaehnemann) [reply recipes/prosic/meta.yaml]: Jep, just trial-and-error. And at the heart of it, this is a `cargo` error (see the [build failure](https://circleci.com/gh/bioconda/bioconda-recipes/79308)) -- so maybe cargo uses clang in the background and thus doesn't find `assert.h` without `clangdev?

[the remark this answers is not in the document]
Comment (dpryan79) [reply recipes/prosic/meta.yaml]: Could be, I don't pretend to be familiar with rust.

**Pull request merged**